shell/platform/darwin/ios/ios_gl_context.mm

```diff
@@ -44,20 +45,23 @@
   }
 }
 
-IOSGLContext::~IOSGLContext() = default;
+fml::WeakPtr<IOSGLContext> IOSGLContext::GetWeakPtr() {
+  return weak_factory_->GetWeakPtr();
+}
 
-std::unique_ptr<IOSGLRenderTarget> IOSGLContext::CreateRenderTarget(
-    fml::scoped_nsobject<CAEAGLLayer> layer) {
-  return std::make_unique<IOSGLRenderTarget>(std::move(layer), context_.get(),
-                                             resource_context_.get());
+bool IOSGLContext::BindRenderbufferStorage(NSUInteger target,
```

Comment:
The `target` argument is not used. Presumably, this was meant to be `GL_RENDERBUFFER`. If this is always going to be the case, can we rid of it in the signature?